components/Card/CardList.tsx

```diff
@@ -15,8 +15,8 @@ function CardList({
   folderListData,
 }: CardListProps): ReactNode {
   return (
-    LinksData?.data &&
-    LinksData.data.map((link) => {
+    LinksData &&
```

Comment:
```suggestion
    LinksData?.map((link) => {
```
이렇게 optional chaining을 잘 활용하면 코드가 훨씬 깔끔하게 보이게 만들수있어요.